server.js

```diff
@@ -1,38 +1,155 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import mongoose from "mongoose";
+import booksData from "./data/books.json";
 
 // If you're using one of our datasets, uncomment the appropriate import below
 // to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
+
+// Database setup:
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+// Mongoose model setup:
+// Book Model
+const Book = mongoose.model("Book", {
+  bookID: {
+    type: Number,
+  },
+  title: {
+    type: String,
+  },
+  authors: {
+    type: String,
+  },
+  average_rating: {
+    type: Number,
+  },
+  isbn: {
+    type: String,
+  },
+  isbn13: {
+    type: String,
+  },
+  language_code: {
+    type: String,
+  },
+  num_pages: {
+    type: Number,
+  },
+  ratings_count: {
+    type: Number,
+  },
+  text_reviews_count: {
+    type: Number,
+  },
+});
```

Comment:
Model created this way allows the user to add empty records to your database. 
Use `required: true` in some fields like title and authors to make it safe.

And one suggestion: mongoose generates unique _ID field for you. Do you really need to add bookID from the data file? I think it will be difficult to increment when you will need to add new records in the future :)